pegasus/sites.v3/code.org/public/dance.haml

```diff
@@ -45,7 +45,23 @@ theme: responsive
   =view :dance_videos
 
 - if request.language == 'en'
+
+  %link{href: "/css/full-resource-block.css", rel: "stylesheet"}
```

Comment:
...because using `student.css` here would overwrite styling used in the other blocks on the page.